nextcord/channel.py

```diff
@@ -1576,6 +1577,10 @@ def permissions_for(self, obj: Union[Member, Role], /) -> Permissions:
             base.value &= ~denied.value
         return base
 
+    def is_nsfw(self) -> bool:
+        """:class:`bool`: Checks if the channel is NSFW."""
+        return self.nsfw
+
```

Comment:
Would it not be easier here to just have `self.nsfw` be the reference people use? Or to rename `self.nsfw` to `self.is_nsfw`

Reply:
I'm not too sure why it is like this either, I followed the same format as other channels though but that sounds like something that should change in the future since it seems unnecessary right now.